### DISCUSSION TITLE
Add a system of getting medals in the game

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15541] [Ideas] [answered]
I propose to develop a system that would show the medals just received in the game.
![ScreenShot_20211109195441](https://user-images.githubusercontent.com/86934170/140971343-5a623ac5-75a1-42f2-8b5f-d4a0cc636a42.jpg)

**Comment by peppy** [comment, marked as answer]
the medal display screen is already implemented, but medal awarding is not yet implemented in lazer. it will come with time.
